src/server/worldserver/worldserver.conf.dist

```diff
@@ -2561,7 +2561,7 @@ BattleGround.PremadeGroupWaitForMatch = 1800000
 #        Default:     0 - (Disabled)
 #                     1 - (Enabled)
 
-Battleground.GiveXPForKills = 0
+Battleground.GiveXPForKills = 1
```

Comment:
This shouldn't be modified, since it changes the original behaviour of the game.